german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Bei meinem Argument ging es mir weniger um den konkreten Begriff, sondern um die generellen Auswahlkriterien. Ich nehme lieber ein Wort, dass einer breiten Masse bekannt ist, als ein vielleicht minimal treffenderes, aber exotisches Wort oder Fachbegriff. Vor allem, wenn sich das Produkt nicht unbedingt an eingefleischte langjährige Extremrollenspieler richten *müsste* (ein Bekannter von mir setzt z.B. ganz konkret TBH in der Sonderpädagogik ein - die Schüler **müssen** rollenspielen... :-)).

Also "Festes Wams" einfach zu "Wams"? Oder noch ein Adjektiv behalten - Festes Wams, Gepolstertes Wams etc.?